spec/models/user_spec.rb

```diff
@@ -25,7 +25,7 @@
 
     it 'marks preorder#status as school_contacted' do
```

Comment:
if the expected value has changed, can you update the example description to match?

Reply:
Yes, good catch, fixed and force pushed.